apps/andi/lib/andi/input_schemas/datasets/query_param.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Andi.InputSchemas.Datasets.QueryParam do
```

Comment:
This seems to be identical to Headers. Could Technical have two lists of the same type of child?

Reply:
I think the idea is that we can store them separately in ecto so we can query against them without complicating the queries and get calls. But you're right, there's a lot of duplication in here that I'll try to clean up